tensorflow/lite/micro/examples/network_tester/input_data.h

```diff
@@ -16,8 +16,93 @@ limitations under the License.
 #ifndef TENSORFLOW_LITE_MICRO_EXAMPLES_NETWORK_TESTER_INPUT_DATA_H_
 #define TENSORFLOW_LITE_MICRO_EXAMPLES_NETWORK_TESTER_INPUT_DATA_H_
 
-static const int input_data_len = 16;
-static const unsigned char input_data[1][16] = {
-    {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16}};
+static const int input_data_len = 1024;
+static const unsigned char input_data[] = {
```

Comment:
is this change part of the current PR?

Reply:
Yes the original model is a uint8 one. So we need an int8 model instead. It is changed from a uint8 maxpool operator to an int8 depthwise operator.

Reply:
So therefore I needed to change the default input data.